lib/mayaUsd/fileio/utils/readUtil.h

```diff
@@ -42,6 +42,16 @@ struct UsdMayaReadUtil
     MAYAUSD_CORE_PUBLIC
     static bool ReadFloat2AsUV();
 
+    /// Returns the imported name of a UV set at \p uvIndex according to the renaming rule and the
```

Comment:
This function description does not seem accurate and refer to a wrong parameter name.

Reply:
Oh, excellent catch - I changed the function signature but forgot to change the comment - I'll fix@